sass/includes/_event-hero.scss

```diff
@@ -0,0 +1,74 @@
+@use "sass:math";
+@use "@nationalarchives/frontend/nationalarchives/variables/grid";
+@use "@nationalarchives/frontend/nationalarchives/tools/media";
+
+.event-hero {
+    background-color: $color__maroon;
+    color: $color__white;
+    display: flex;
+    flex-direction: column;
+
+    @include media.on-larger-than-mobile {
+        flex-direction: row;
+        justify-content: flex-end;
+    }
+
+    &__breadcrumb-mobile {
+        @include media.on-larger-than-mobile {
+            display: none;
+        }
+    }
+
+    &__breadcrumb-desktop {
+        display: none;
+
+        @include media.on-larger-than-mobile {
+            display: block;
+        }
+    }
+
+    &__info-wrapper {
+        padding-bottom: 1.5rem;
+        order: 2;
+
+        @include media.on-larger-than-mobile {
+            order: 0;
+            // allows the text to fill half the width that a tna-container wrapper would create
+            max-width: math.div(grid.$largest-container-width, 2);
+            width: 50%;
+            padding-bottom: 7.812rem;
+        }
+    }
+
+    &__image-wrapper {
+        order: 1;
+        position: relative;
+        margin-bottom: 1.5rem;
+
+        @include media.on-larger-than-mobile {
+            order: 0;
+            min-height: 560px;
+            flex-basis: 50%;
+            width: 50%;
+            margin-bottom: 0;
+        }
+    }
+
+    &__picture {
+        @include media.on-larger-than-mobile {
+            position: absolute;
+            inset: 0;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        height: auto;
+
+        @include media.on-larger-than-mobile {
```

Comment:
Could we group all these media queries at the end of the component else we'll end up with a lot of repetition of `@media` rules in the final CSS.

E.g.

```scss
.event-hero {
    &__picture { /* ... */ }

    &__image { /* ... */ }

    @include media.on-larger-than-mobile {
        &__picture { /* ... */ }

        &__image { /* ... */ }
    }
}
```

Reply:
Sure. Happy to go with whatever is your normal convention. At torchbox we always group media queries with the element they relate to to aid clarity when reading the code, but I know opinions differ on that front.